Source/Core/DolphinQt2/GameList/ListProxyModel.cpp

```diff
@@ -34,3 +34,9 @@ QVariant ListProxyModel::data(const QModelIndex& i, int role) const
   }
   return QVariant();
 }
+
+bool ListProxyModel::filterAcceptsRow(int sourceRow, const QModelIndex& sourceParent) const
```

Comment:
These parameter names should be `source_row` and `source_parent` to match Dolphin's coding style (and in the TableProxyModel class as well).